.TH "Renci.SshNet.Common.SshAuthenticationException" 3 "Fri Jul 5 2013" "Version 1.0" "HSA.InfoSys" \" -*- nroff -*-
.ad l
.nh
.SH NAME
Renci.SshNet.Common.SshAuthenticationException \- 
.PP
The exception that is thrown when authentication failed\&.  

.SH SYNOPSIS
.br
.PP
.PP
Inherits \fBRenci\&.SshNet\&.Common\&.SshException\fP, and \fBRenci\&.SshNet\&.Common\&.SshException\fP\&.
.SS "Public Member Functions"

.in +1c
.ti -1c
.RI "\fBSshAuthenticationException\fP ()"
.br
.RI "\fIInitializes a new instance of the \fBSshAuthenticationException\fP class\&. \fP"
.ti -1c
.RI "\fBSshAuthenticationException\fP (string message)"
.br
.RI "\fIInitializes a new instance of the \fBSshAuthenticationException\fP class\&. \fP"
.ti -1c
.RI "\fBSshAuthenticationException\fP (string message, Exception innerException)"
.br
.RI "\fIInitializes a new instance of the \fBSshAuthenticationException\fP class\&. \fP"
.in -1c
.SS "Protected Member Functions"

.in +1c
.ti -1c
.RI "\fBSshAuthenticationException\fP (SerializationInfo info, StreamingContext context)"
.br
.RI "\fIInitializes a new instance of the \fBSshAuthenticationException\fP class\&. \fP"
.in -1c
.SH "Detailed Description"
.PP 
The exception that is thrown when authentication failed\&. 


.PP
Definition at line 11 of file SshAuthenticationException\&.cs\&.
.SH "Constructor & Destructor Documentation"
.PP 
.SS "Renci\&.SshNet\&.Common\&.SshAuthenticationException\&.SshAuthenticationException ()"

.PP
Initializes a new instance of the \fBSshAuthenticationException\fP class\&. 
.PP
Definition at line 16 of file SshAuthenticationException\&.cs\&.
.SS "Renci\&.SshNet\&.Common\&.SshAuthenticationException\&.SshAuthenticationException (stringmessage)"

.PP
Initializes a new instance of the \fBSshAuthenticationException\fP class\&. 
.PP
\fBParameters:\fP
.RS 4
\fImessage\fP The message\&.
.RE
.PP

.PP
Definition at line 25 of file SshAuthenticationException\&.cs\&.
.SS "Renci\&.SshNet\&.Common\&.SshAuthenticationException\&.SshAuthenticationException (stringmessage, ExceptioninnerException)"

.PP
Initializes a new instance of the \fBSshAuthenticationException\fP class\&. 
.PP
\fBParameters:\fP
.RS 4
\fImessage\fP The message\&.
.br
\fIinnerException\fP The inner exception\&.
.RE
.PP

.PP
Definition at line 36 of file SshAuthenticationException\&.cs\&.
.SS "Renci\&.SshNet\&.Common\&.SshAuthenticationException\&.SshAuthenticationException (SerializationInfoinfo, StreamingContextcontext)\fC [protected]\fP"

.PP
Initializes a new instance of the \fBSshAuthenticationException\fP class\&. 
.PP
\fBParameters:\fP
.RS 4
\fIinfo\fP The T:System\&.Runtime\&.Serialization\&.SerializationInfo that holds the serialized object data about the exception being thrown\&.
.br
\fIcontext\fP The T:System\&.Runtime\&.Serialization\&.StreamingContext that contains contextual information about the source or destination\&.
.RE
.PP
\fBExceptions:\fP
.RS 4
\fIT:System\&.ArgumentNullException\fP The \fIinfo\fP  parameter is null\&. 
.RE
.PP
.PP
\fBExceptions:\fP
.RS 4
\fIT:System\&.Runtime\&.Serialization\&.SerializationException\fP The class name is null or P:System\&.Exception\&.HResult is zero (0)\&. 
.RE
.PP

.PP
Definition at line 23 of file SshAuthenticationException\&.NET40\&.cs\&.

.SH "Author"
.PP 
Generated automatically by Doxygen for HSA\&.InfoSys from the source code\&.
